app/views/static_pages/about.html.erb

```diff
@@ -1,12 +1,7 @@
 <% provide(:title, t(:about)) %>
-<h1><%= t(:about) %></h1>
+<h2><%= t(:about) %></h2>
 <div class="search-blocks search-blocks-top-orange">
   <div class="col-md-8 col-xs-12">
-    <%= simple_format(t(:about_us_first))%>
-    <%= simple_format(t(:about_us_second))%>
-    <%= simple_format(t(:about_us_third))%>
-  </div>
-  <div class="col-md-4 col-xs-12">
-    <%= render Menu.where(location: 'guild') %>
+    Nu har vi äntligen ett digitalt valsystem!
```

Comment:
I agree, but this is just a placeholder :) We are probably going to change this page anyway